command/operator_rekey.go

```diff
@@ -58,6 +63,11 @@ Usage: vault operator rekey [options] [KEY]
   the command. If key is specified as "-", the command will read from stdin. If
   a TTY is available, the command will prompt for text.
 
+  If the flag -target=recovery is supplied, then this operation will
```

Comment:
You might want to add something to the effect that: in this mode, recovery keys are also used in place of the unseal keys as the inputs.  You've clarified this for the outputs, but it's left ambiguous as to the inputs.